src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -1828,6 +1828,40 @@ final void initResettableValues() {
         sqlWarnings = null;
         sCatalog = originalCatalog;
         databaseMetaData = null;
+
+        try{
+            // check QUOTED_IDENTIFIER property
+            String quotedIdentifierProperty = SQLServerDriverStringProperty.QUOTED_IDENTIFIER.toString();
+            String quotedIdentifierValue = activeConnectionProperties.getProperty(quotedIdentifierProperty);
+            if (null == quotedIdentifierValue) {
+                quotedIdentifierValue = SQLServerDriverStringProperty.QUOTED_IDENTIFIER.getDefaultValue();
+                activeConnectionProperties.setProperty(quotedIdentifierProperty, quotedIdentifierValue);
+            }
+
+            String quotedIdentifierOption = OnOffOption.valueOfString(quotedIdentifierValue).toString();
+            if (quotedIdentifierOption.compareToIgnoreCase(OnOffOption.OFF.toString()) == 0) {
+                connectionCommand("SET QUOTED_IDENTIFIER OFF", "quotedIdentifier");
+            } else if (quotedIdentifierOption.compareToIgnoreCase(OnOffOption.ON.toString()) == 0) {
+                connectionCommand("SET QUOTED_IDENTIFIER ON", "quotedIdentifier");
+            }
+
+            // check CONCAT_NULL_YIELDS_NULL property
+            String concatNullYieldsNullProperty = SQLServerDriverStringProperty.CONCAT_NULL_YIELDS_NULL.toString();
+            String concatNullYieldsNullValue = activeConnectionProperties.getProperty(concatNullYieldsNullProperty);
+            if (null == concatNullYieldsNullValue) {
+                concatNullYieldsNullValue = SQLServerDriverStringProperty.CONCAT_NULL_YIELDS_NULL.getDefaultValue();
+                activeConnectionProperties.setProperty(concatNullYieldsNullProperty, concatNullYieldsNullValue);
+            }
+            String concatNullYieldsNullOption = OnOffOption.valueOfString(concatNullYieldsNullValue).toString();
+            if (concatNullYieldsNullOption.compareToIgnoreCase(OnOffOption.OFF.toString()) == 0) {
+                connectionCommand("SET CONCAT_NULL_YIELDS_NULL OFF", "concatNullYieldsNull");
+
+            } else if (concatNullYieldsNullOption.compareToIgnoreCase(OnOffOption.ON.toString()) == 0) {
+                connectionCommand("SET CONCAT_NULL_YIELDS_NULL ON", "concatNullYieldsNull");
+            }
```

Comment:
(Verify) Similarly, this is the default state and we don't need to send ON.

Reply:
Yes, server side it is default.

SET CONCAT_NULL_YIELDS_NULL OFF and the CONCAT_NULL_YIELDS_NULL OFF database option are deprecated. Starting with SQL Server 2017 (14.x), CONCAT_NULL_YIELDS_NULL is always set to ON. Deprecated features shouldn't be used in new applications. 

Ref: https://learn.microsoft.com/en-us/sql/t-sql/statements/set-concat-null-yields-null-transact-sql?view=sql-server-ver16